modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -32,77 +32,154 @@ on {rh-openstack-first}
 endif::[]
 to use an external load balancer in place of the default load balancer.
 
-You can also configure an {product-title} cluster to use an external load balancer that supports multiple subnets. If you use multiple subnets, you can explicitly list all the IP addresses in any networks that are used by your load balancer targets. This configuration can reduce maintenance overhead because you can create and destroy nodes within those networks without reconfiguring the load balancer targets.
+[IMPORTANT]
+====
+Configuring an external load balancer depends on a specific vendor's load balancer. 
```

Comment:
I think this might read better if it said "your specific vendor's load balancer" 

and consult your vendors info...etcetera